webproject/src/main/java/com/google/sps/servlets/TextMessageServlet.java

```diff
@@ -0,0 +1,71 @@
+package com.google.sps.servlets;
+
+import com.google.sps.authentication.AuthenticationHandler;
+import com.google.appengine.api.users.User;
+import com.google.sps.data.Message;
+import com.google.firebase.FirebaseApp;
+import com.google.firebase.FirebaseOptions;
+import com.google.auth.oauth2.GoogleCredentials;
+import com.google.firebase.database.FirebaseDatabase;
+import com.google.appengine.api.blobstore.BlobInfo;
+import com.google.appengine.api.blobstore.BlobInfoFactory;
+import com.google.appengine.api.blobstore.BlobKey;
+import com.google.appengine.api.blobstore.BlobstoreService;
+import com.google.appengine.api.blobstore.BlobstoreServiceFactory;
+import com.google.appengine.api.images.ImagesService;
+import com.google.appengine.api.images.ImagesServiceFactory;
+import com.google.appengine.api.images.ServingUrlOptions;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.util.List;
+import java.util.Map;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+public class TextMessageServlet extends HttpServlet {
+    private static String username;
+
+    @Override
+    public void init() {
+        try {
+            // Fetch the service account key JSON file contents
```

Comment:
we need to clean this later, there are 3 other places which do the same thing as this

Reply:
Understood!